server/graphql.server.ts

```diff
@@ -55,4 +60,23 @@ export class PolarisGraphQLServer {
         let endpoint = properties['endpoint'];
         this._polarisProperties = new PolarisProperties(port, endpoint);
     }
+
+    public getApplicationProperties(applicationProperties):ApplicationLogProperties{
+        if (applicationProperties!= null)
+            return new ApplicationLogProperties(applicationProperties.id, applicationProperties.name,
+                 applicationProperties.repositoryVersion, applicationProperties.environment,applicationProperties.component);
+        return new ApplicationLogProperties("p01aris-10gs", "polaris-logs", "v1", "dev", "component");
+    }
+
+    public getLogPath(logPath:string):string{
+        console.log(logPath);
+        if (logPath != null)
+            return logPath;
+        else
+            return "log-file.log";
+    }
+
```

Comment:
I do not like the methods that their only purpose is to define default values for the config properties. The config should be injected with values, whether they are default or user defined. Check how you can do it, maybe wrap the config itself. And then you can remove the getLogPath getApplicationProperties methods